prow/presubmit.yaml

```diff
@@ -150,6 +150,14 @@ kubernetes/kubernetes:
   trigger: "@k8s-bot (node )?(e2e )?test this"
 
 - name: pull-kubernetes-e2e-gce-cri
+   skip_branches:
```

Comment:
> I would suggest use branches: master to whitelist master rather than blackout all the release branches :-)

Does that mean I'll have to change this once we cut the 1.6 branch? :-)